node/src/blockchain/errors/internal_errors.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2025, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Debug, PartialEq, Clone, Eq)]
+pub enum InternalError {
```

Comment:
I think this should be called `MempoolError`. In case you change this, change the `Kind` enum too.